UI/StressLabel.gd

```diff
@@ -0,0 +1,14 @@
+extends Label
+## Simple setter that pulls from the stress component.
+
+
+@export var stress_component: StressComponent = null
+
+
+func _ready() -> void:
+	stress_component.on_stress_changed.connect(_set_stress)
+	_set_stress(stress_component.current_stress)
+
+
+func _set_stress(new_stress: float) -> void:
+	text = str(new_stress) + " / " + str(stress_component.max_stress)
```

Comment:
Does fully soothing an enemy cause the enemy to stop generating stress? If so, should we have logic here to update the text to like "Soothed" or something of the sort?

Reply:
yes, as you can see in the `on_turn_start` function of `StressComponent`, we only generate stress if the stress is above 1
https://github.com/Saplings-Projects/1M_sub/blob/2139919b0ed0c583839aa1af2ea8d19e87374753/Entity/Components/StressComponent.gd#L91-L94

I can change the value on display when the entity is soothed without problem